docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -32,53 +32,83 @@ This function is useful for removing global state or closing a server that was u
-and provides access to `page`, `params`, `browser`, and `context`:
-
-* `browser` -- A {playwright-api-docs}[browser] object created by Playwright.
-* `page` -- A https://playwright.dev/docs/api/class-page[page] object from Playwright that lets you control the browser's current page.
-* `context` -- A https://playwright.dev/docs/api/class-browsercontext[browser context] that doesn't share cookies or cache with other browser contexts.
-* `params` -- User-defined variables that allow you to invoke the Synthetics suite with custom parameters.
-For example, if you want to use a different homepage depending on the `env`
-(localhost for `dev`, and a URL for `prod`). See <<synthetics-params-secrets>>
-for more information.
+A journey tests one discrete unit of functionality.
+For example, logging into a website, adding something to a cart, or joining a mailing list.
 
-Putting it all together, a basic, two step journey might look something like this:
+Start by creating a new file called using the `.journey.ts` or `.journey.js` file extension,
+importing the synthetics library, and adding a new journey.
 
 [source,js]
 ----
-journey("Journey name", ({ page }) => {
-    step("Step 1 name", () => {
-      // Do something here
-    })
-    step("Step 2 name", () => {
-      // Do something else here
-    })
+journey('Journey name', ({ page, browser, context, params }) => {
+  // Add steps here
 });
 ----
 
+The `journey` function takes a single parameter, `name` ('Journey name' in the example above).
+Then you pass a callback to the global `journey` function to encapsulate the whole test.
+This callback function provides access to fresh Playwright `page`, `params`, `browser`, and `context` instances.
+
+[horizontal]
+`name`::        A string you provide to describe the journey.
+`page`::        A https://playwright.dev/docs/api/class-page[page] object from Playwright
+                that lets you control the browser's current page.
+`browser`::     A {playwright-api-docs}[browser] object created by Playwright.
+`context`::     A https://playwright.dev/docs/api/class-browsercontext[browser context] 
+                that doesn't share cookies or cache with other browser contexts.
+`params`::      User-defined variables that allow you to invoke the Synthetics suite with custom parameters.
+                For example, if you want to use a different homepage depending on the `env`
+                (localhost for `dev`, and a URL for `prod`). See <<synthetics-params-secrets>>
+                for more information.
+
+
 [discrete]
 [[synthetics-create-step]]
-=== Create a `step`
+=== Add steps
 
-Steps can be as simple or complex as you need them to be.
-The {playwright-api-docs}[Playwright page API reference] will be your friend while writing tests.
+A journey consists of multiple _steps_. Steps are actions that should be completed in a specific order.
+Steps are displayed individually in the {uptime-app} for easy debugging and error tracking.
```

Comment:
Maybe worth mentioning that each step gets a screenshot. I think that's very helpful for users to do debugging.